.github/codecov.yml

```diff
@@ -1,12 +1,4 @@
 coverage:
   status:
-    project:
-      default:
-        threshold: 50%
-        base: development
-    patch:
-      default:
-        threshold: 50%
-    changes:
-      default:
-        threshold: 10%
+    project: off
+    patch: off
```

Comment:
We can revisit how we approach codecoverage in the project later. I don't like having to ignore red fails in the CI - better to admit reality and turn off the check.